helm-chart/renku-gateway/templates/deployment.yaml

```diff
@@ -28,6 +32,75 @@ spec:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
+            - name: admin
+              containerPort: 8080
+          args:
+            - --configfile=/config/traefik.toml
+          volumeMounts:
+            - mountPath: /config
+              name: config
+          readinessProbe:
+            tcpSocket:
+              port: {{ .Values.service.port }}
+            failureThreshold: 1
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 2
+          livenessProbe:
+            tcpSocket:
+              port: {{ .Values.service.port }}
+            failureThreshold: 3
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 2
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+    {{- with .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    {{- with .Values.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "gateway.fullname" . }}-auth
+  labels:
+    app: {{ template "gateway.name" . }}-auth
+    chart: {{ template "gateway.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    {{ include "call-nested" (list . "redis" "redis.fullname") }}-client: "true"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "gateway.name" . }}-auth
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "gateway.name" . }}-auth
+        release: {{ .Release.Name }}
+        {{ include "call-nested" (list . "redis" "redis.fullname") }}-client: "true"
```

Comment:
@ableuler I forgot to move this label. Does it work for you now?

Reply:
did you turn off the redis network policy also?

Reply:
where?

Reply:
The reason that line is there is because the redis network policy only allows connections from pods with that label afaik

Reply:
See https://github.com/SwissDataScienceCenter/renku-gateway/blob/master/helm-chart/renku-gateway/values.yaml

Reply:
It was my mistake as I moved it to metadata label not pod label.

Reply:
Aha! Sorry, I missed that part :)

Reply:
@jirikuncar No, still doesn't work. The gw still tries to connect to its own redis, but no such service is being created anymore. Just compare the services under the namespaces `andreas` and `renku` currently deployed on the dev cluster. Did you rip out the second redis on purpose?